samples/manager/manager.js

```diff
@@ -199,27 +199,33 @@ const createDevice = async (
     // optional auth parameters.
   });
 
-  const regPath = iotClient.registryPath(projectId, cloudRegion, registryId);
-  const device = {
-    id: deviceId,
-    credentials: [
-      {
-        publicKey: {
-          format: publicKeyFormat,
-          key: fs.readFileSync(publicKeyFile).toString(),
+  async function createDevice() {
+    // Construct request
+    const regPath = iotClient.registryPath(projectId, cloudRegion, registryId);
+    const device = {
+      id: deviceId,
+      credentials: [
+        {
+          publicKey: {
+            format: publicKeyFormat,
+            key: fs.readFileSync(publicKeyFile).toString(),
+          },
         },
-      },
-    ],
-  };
+      ],
+    };
+
+    const request = {
+      parent: regPath,
+      device,
+    };
 
-  const request = {
-    parent: regPath,
-    device,
-  };
-  try {
     const responses = await iotClient.createDevice(request);
     const response = responses[0];
     console.log('Created device', response);
+  }
+
+  try {
+    createDevice();
   } catch (err) {
```

Comment:
This `catch` doesn't really do much.  Better to remove the try/catch and just let this throw!

Reply:
Ok I will apply this change to all samples of IoT